### ISSUE TITLE
[Bug] Manufacturer Info set to Default

### ISSUE THREAD

**Issue by mitch7391** [ztalbot2000/homebridge-cmd4#125]
**Describe The Bug:**
Hey John, I have noticed in the last few weeks and have only had the time to bring it up now; but it appears that the `Manufacturer`, `SerialNumber` and `Model` for all my accessories shows as `Default-Manufacturer`, `Default-SerialNumber` and `Default-Model`. They used to show as what I have defined in my `config.json`, but no longer seem to.

**Expected Behaviour:**
Should display what I have set in my `config.json`.

**Cmd4 Config:**
Example of one accessory (you know how my config looks):

```json

                {
                    "type": "Switch",
                    "displayName": "Living",
                    "on": "FALSE",
                    "name": "Living",
                    "manufacturer": "Advantage Air Australia",
                    "model": "e-zone",
                    "serialNumber": "Fujitsu e-zone2",
                    "queue": "A",
                    "polling": true,
                    "state_cmd": "'/usr/local/lib/node_modules/homebridge-cmd4-advantageair/AdvAir.sh'",
                    "state_cmd_suffix": "z01 ${IP}"
                }
```

**Screenshots:**
![41355A4E-E2B5-4656-A202-8ADBAB541BC0](https://user-images.githubusercontent.com/40288237/151806763-79f233c7-4263-44f8-b979-3e95c2bd9cdf.jpeg)

**Environment:**

* **Node.js Version**: v16.13.2
* **NPM Version**: v8.1.2
* **Homebridge Version**: 1.4.0
* **homebridge-cmd4 Version**: v6.1.0
* **Operating System**: Raspbian
* **Process Supervisor**: hb-service

**Comment by ztalbot2000** [comment]
Hi Mitch,

Weird.  Nothing has changed, but I can have a look.

Ttyl,
John

**Comment by mitch7391** [comment]
I have tried doing a cache delete and let Homebridge create them new; but still nothing. Definitely a strange one indeed!

**Comment by ztalbot2000** [comment]
Hi Mitch,

So Cmd4 is returning the expected values. What is being displayed though is
incorrect.  This might be a homebridge issue.  Still poking.

Ttyl,
John

**Comment by mitch7391** [comment]
> Do not know when that happened either.

I think I first noticed it a few cmd4 versions ago, but it was very low on my radar at the time and forgot about it.

> This might be a homebridge issue.  Still poking.

Very interesting! Keen to hear what you find!

**Comment by ztalbot2000** [comment]
Hi Mitch,

  So Eve is showing the correct information. So it seems to be neither a
homebridge or Cmd4 issue.  I will say the problem must be the Home App.
Have you noticed any other plugins with the same issue?

Ttyl,
John

**Comment by mitch7391** [comment]
Hmmm… It appears all my other Homebridge and native HomeKit accessories are working fine. My Hisense TV through our cmd4 work and my Advantage Air accessories are not working. My PS5 added as a switch by itself through cmd4 is partially working!

![9A7E2F61-359D-45CB-95E6-E7B184DBB0FF](https://user-images.githubusercontent.com/40288237/151821564-0ae6f4ca-ae24-4de7-9871-9d310a6fab15.jpeg)

**Comment by ztalbot2000** [comment]
Hi Mitch,

Thanks for checking.  Something is right and something is wrong. Maybe
something stupid like spaces or dashes.  I'll poke and find out.

Ttyl,
John

**Comment by mitch7391** [comment]
Perhaps, but `Model` in my Advantage Air config does not use spaces and is still not appearing...

```shell

                {
                    "type": "Switch",
                    "displayName": "Living",
                    "on": "FALSE",
                    "name": "Living",
                    "manufacturer": "Advantage Air Australia",
                    "model": "e-zone",
                    "serialNumber": "Fujitsu e-zone2",
                    "queue": "A",
                    "polling": true,
                    "state_cmd": "'/usr/local/lib/node_modules/homebridge-cmd4-advantageair/AdvAir.sh'",
                    "state_cmd_suffix": "z01 ${IP}"
                }
```

**Comment by ztalbot2000** [comment]
Hi Mitch,

It does use a dash though.  I'll try different sequences.

Ttyl,
John

**Comment by mitch7391** [comment]
Very fair point!

**Comment by ztalbot2000** [comment]
Hi Mitch,

So I changed a switches manufacturer to "zarfLand". Eve picks it up, but
HomeKit does not.

Still poking.

John

**Comment by ztalbot2000** [comment]
Hi Mitch,

   I keep hitting the same wall. Eve is fine. Home is not. Most things in
Home do not have the manufacturer, no matter even if it's a one word string.
My Aircon lost its. Manufacturer info as well in Home.
This one is tricky as nothing has changed on our end for a long time.

Ttyl,
John

**Comment by mitch7391** [comment]
Damn, makes me wish I had brought it up sooner haha oh well... It is my first day off tomorrow, so when I have had my partial sleep out of night shift I will do a little trial and error to see if I can draw some connections.

**Comment by ztalbot2000** [comment]
I think I found the issue.  My fix worked I'll ship it out in minute

Ttyl,
John

**Comment by mitch7391** [comment]
Ooooo sounds like a winner :D

**Comment by ztalbot2000** [comment]
Hi Mitch,

  Just release v6.1.1

Enjoy,
John

**Comment by mitch7391** [comment]
Love your work as always John :) thanks for getting to this so fast!

**Comment by mitch7391** [comment]
What was the issue in the end..?

**Comment by ztalbot2000** [comment]
Hi Mitch,

  This has been there for awhile. I never noticed it because as you had
seen, sometimes it works and it always worked in Eve.
During clean up I separated characteristics from Cmd4 directives.
The manufacturer, serialNumber and model are kind of both. Not parsing them
in both manners, eliminated them from showing up in Home for some reason
and in some cases.
Once I parsed for them, it worked immediately.

Ttyl,
John
